Signal/src/view controllers/MessageComposeTableViewController.m

```diff
@@ -56,7 +75,8 @@ - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder
     }
 
     _contactsManager = [Environment getCurrent].contactsManager;
-
+    _phoneNumberAccountSet = [NSMutableSet set];
```

Comment:
Not that we're currently using the naked `init` but if we did someone would be in for a rude surprise to find this is `nil`